src/test/java/com/placelab/tests/LoginTestPasswordEmpty.java

```diff
@@ -0,0 +1,63 @@
+package com.placelab.tests;
+
+import com.placelab.utils.WebDriverSetup;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebDriver;
+import org.testng.Assert;
+import org.testng.annotations.AfterTest;
+import org.testng.annotations.BeforeTest;
+import org.testng.annotations.Parameters;
+import org.testng.annotations.Test;
+
+public class LoginTestPasswordEmpty {
+    private WebDriver driver;
+
+    @Parameters("browser")
+    @BeforeTest
+    public void setup(final String browser) {
+        driver = WebDriverSetup.getWebDriver(browser);
+        driver.get("https://demo.placelab.com/");
+
+        System.out.println("Opened browser: " + browser);
+    }
+
+    @Parameters("email")
+    @Test
+    public void testPositiveLogin(final String email) {
```

Comment:
update method names accordingly, this one suggest that the test is for the positive test steps